ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -882,7 +871,7 @@ FEDataManager::spread(const int f_data_idx,
                     IBTK::get_nodal_dof_indices(F_dof_map, n, i, F_idxs);
                     for (const auto& F_idx : F_idxs)
                     {
-                        F_dX_node.push_back(F_dX_local_soln[F_dX_petsc_vec->map_global_to_local_index(F_idx)]);
+                        F_x_dX_node.push_back(F_x_dX_local_soln[F_x_dX_petsc_vec->map_global_to_local_index(F_idx)]);
                     }
```

Comment:
These loops (over n_vars then over nodes per element) look wrong to me - we expect the input to the spread function to have the index over vars running fastest (i.e., we have `REAL X(0:NDIM-1,0:*)` in Fortran). We definitely use 'vars fastest' in the nonnodal case (see `sum_weighted_elem_solution`).

Reply:
Note that we are in fact looping over in this order, because at each node we then loop over the variables (which for vector-valued quantities correspond to the components).

Reply:
Oh, I see - that makes sense.